src/commandHandlers/bio.ts

```diff
@@ -0,0 +1,66 @@
+import { MessageEmbed, Message } from 'discord.js';
+
+import config from '../config';
+import { db } from '../firebase';
+
+/**
+ * This command lets the user set their personal data if others want to follow them on other social platforms
+ */
+export const command = async (arg: string, embed: MessageEmbed, message: Message) => {
+    const args = arg.split('||');
+    const mention = message.mentions.users.first();
+
+    if (args[0].length && !mention && !config.BIO.includes(args[0].trim())) {
+        embed
+        .setTitle('Edit Bio (error)')
+        .setDescription(`Bio option not valid, please use one of the following: ${config.BIO.join(', ')}`)
+        .addField('ERROR', 'Invalid argument')
+        .addField('Usage', usage)
+
+        return embed;
+    }
+
+    // get information
+    if (!args[0].length || (!args[1] && mention)) {
+        embed.setDescription('Reading bio');
+        const doc = await db
+            .collection('users')
+            .doc(mention ? mention!.id : message.author.id)
+            .get();
+
+        if (!doc.data()) {
+            embed.addField('Description', 'No bio details found');
+        }
+
+        if (doc.data()) {
```

Comment:
do we need this `if`? Since the `if` before already checks if it's falsy, I think we can remove this `if`.
Also, not sure what the method `data` does, but maybe we can store the value on a variable, and only call the method once, but maybe it's not necessary to do that?

Reply:
- We could store the function call to a variable 👍 
- Not sure how `Object.entries` would react if it is undefined, I can try

Reply:
I can do an `if/else`, let me push the changes and see what you think